prisma/migrations/20241124172312_rename_column/migration.sql

```diff
@@ -0,0 +1,17 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `shortUrl` on the `link` table. All the data in the column will be lost.
+  - A unique constraint covering the columns `[shortUrlCode]` on the table `Link` will be added. If there are existing duplicate values, this will fail.
+  - Added the required column `shortUrlCode` to the `Link` table without a default value. This is not possible if the table is not empty.
```

Comment:
**issue**: Adding a non-nullable column `shortUrlCode` without a default value will fail if the table is not empty. Consider providing a default value or handling existing rows.